apps/galaxy/src/modules/Orders/OpenOrders/OpenOrdersList.tsx

```diff
@@ -42,16 +67,44 @@ const OpenOrdersList = () => {
     }),
   ];
 
+  const Submit = () => {
```

Comment:
if `Submit` is not a component then this should not be PascalCase. And I would go for `onSubmitSelectedOrders` to indicate the intent of this function here.